orchagent/nexthopkey.h

```diff
@@ -98,6 +117,73 @@ struct NextHopKey
     {
         return (ip_address.getV4Addr() == 0);
     }
+
+    bool isMplsNextHop() const
+    {
+        return (label_stack.getSize() > 0);
+    }
+
+    std::string parse_mpls(const std::string& str)
+    {
+        std::string ip_str;
+        auto keys = tokenize(str, LABELSTACK_DELIMITER);
```

Comment:
here "LABELSTACK_DELIMETER" is used. How would this work?

push+100+<ipstr>

Reply:
Please see latest code commit with requested comment and example formatting.